### PR TITLE
Clean up provided containers

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: seaworthy/containers/postgresql.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        self.password = password
+
+    def base_kwargs(self):
+        return {
+            'environment': {
+                'POSTGRES_DB': self.database,
+                'POSTGRES_USER': self.user,
+                'POSTGRES_PASSWORD': self.password,
+            },
+            'tmpfs': {'/var/lib/postgresql/data': 'uid=70,gid=70'},
```
Comment (JayH5) [inline seaworthy/containers/postgresql.py]: Wondering if these `tmpfs` things belong here or if they should be optional 🤔

Comment (jerith) [reply seaworthy/containers/postgresql.py]: Nah, I think this is fine. If someone specifically wants the data written to disk they can override the container definition.

File: seaworthy/containers/rabbitmq.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
+from seaworthy.logs import output_lines
+from .base import ContainerBase
+
+
+def _parse_rabbitmq_user(user_line):
+    user_tags = user_line.split('\t', 1)
+    if len(user_tags) != 2:  # pragma: no cover
+        raise RuntimeError()
```
Comment (JayH5) [inline seaworthy/containers/rabbitmq.py]: Maybe just delete this check completely? A `RuntimeError` without a message is possibly less use than the regular error we'd get. And this isn't supposed to be a very robust parser.

Comment (jerith) [reply seaworthy/containers/rabbitmq.py]: Good point.

**Pull request merged**